_includes/nav_v1.0.html

```diff
@@ -5,8 +5,8 @@
 	<li class="navContainer">
 		<a href="{{ site.url }}/v1.0/docs/whatisk8s.html" class="nav-link">What is Kubernetes?</a>
 	</li><!-- 
-					GETTING STARTED
-					 -->
+GETTING STARTED
```

Comment:
This file is here: https://github.com/RichieEscarez/kubernetes/blob/syncnavonload/_includes/nav_v1.0.html